examples/index.md

```diff
@@ -162,3 +162,241 @@ Content-Type: application/vnd.api+json
+useful for single errors, to save clients the trouble of consulting the HTTP
+headers, or for using JSON API over non-HTTP protocols, which may be officially
+supported in the near future.
+
+### Multiple Errors
+
+When multiple errors occur in response to a single request, the server
+can simply add each error to the `errors` array:
+
+```http
+HTTP/1.1 400 Bad Request
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "status": "403",
+      "source": { "pointer": "data/attributes/secret-powers" },
+      "detail": "Editing secret powers is not authorized on Sundays."
+    },
+    {
+      "status": "422",
+      "source": { "pointer": "data/attributes/volume" },
+      "detail": "Volume does not, in fact, go to 11."
+    },
+    {
+      "status": "500",
+      "source": { "pointer": "data/attributes/reputation" },
+      "title": "The backend responded with an error",
+      "detail": "Reputation service not responding after three requests."
+    }
+  ]
+}
+```
+
+The only uniqueness constraint on error objects is the `id` field. Thus,
+multiple errors on the same attribute can each be given their own error
+object. The example below shows multiple errors on the `first-name` attribute:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "source": { "pointer": "data/attributes/first-name" },
+      "title": "Invalid Attribute",
+      "detail": "First name must contain at least three characters."
+    },
+    {
+      "source": { "pointer": "data/attributes/first-name" },
+      "title": "Invalid Attribute",
+      "detail": "First name must contain an emoji."
+    }
+  ]
+}
+```
+
+> Note: in the responses above with a 422 status code, `400 Bad Request` would
+also be acceptable. ([http://stackoverflow.com/a/20215807/1261879](More details.))
+JSON API doesn't take a position on 400 vs. 422.
+
+### Error Codes
+
+The `code` member of an error object contains an application-specific code
+representing the type of problem encountered. `code` is similar to `title`
+in that both identify a general type of problem (unlike `detail`, which is
+specific to the particular instance of the problem), but dealing with `code`
+is easier programatically, because the "same" `title` may appear in different
+forms due to localization.
+
+For the example below, imagine the API docs specifed the following mapping:
+
+> | Code | Problem                                                   |
+> |------|-----------------------------------------------------------|
+> |  123 | Value too short                                           |
+> |  225 | Password lacks a letter, number, or punctuation character |
+> |  226 | Passwords do not match                                    |
+> |  227 | Password cannot be one of last five passwords             |
+
+Multiple errors on `password` attribute, with error `code`:
```

Comment:
Yes, quoting the attribute values would be more consistent with the base spec. There, I believe we actually quote _all_ member values. So, probably, we should be quoting `code`, `detail`, `title`, etc. Honestly, though, it's not a big deal (to me anyway) one way or the other.